Solutions/Bitglass/Analytic Rules/BitglassImpossibleTravelDistance.yaml

```diff
@@ -26,5 +26,5 @@ entityMappings:
     fieldMappings:
       - identifier: Name
         columnName: AccountCustomEntity
-version: 1.0.0
+version: 1.0.2

```

Comment:
Can you please downgrade the version to 1.0.1. Let me know if you have added it specifically.